cargo bench failed for kclvm sema

## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

```
cd kclvm/sema
cargo bench
```

### 2. What did you expect to see? (Required)

Successfully build and print benchmark result

### 3. What did you see instead (Required)

```
root@d4f417f1cca2:/workspaces/kcl/kclvm/sema# cargo bench
   Compiling kclvm-sema v0.7.0-alpha.2 (/workspaces/kcl/kclvm/sema)
error[E0308]: mismatched types
  --> sema/benches/my_benchmark.rs:13:21
   |
13 |                     Rc::new(Type::STR),
   |                     ^^^^^^^^^^^^^^^^^^ expected struct `Arc`, found struct `Rc`
   |
   = note: expected struct `Arc<kclvm_sema::ty::Type>`
              found struct `Rc<kclvm_sema::ty::Type>`

error[E0308]: arguments to this function are incorrect
  --> sema/benches/my_benchmark.rs:14:29
   |
14 |                     Rc::new(Type::dict(Rc::new(Type::STR), Rc::new(Type::STR))),
   |                             ^^^^^^^^^^
   |
note: expected struct `Arc`, found struct `Rc`
  --> sema/benches/my_benchmark.rs:14:40
   |
14 |                     Rc::new(Type::dict(Rc::new(Type::STR), Rc::new(Type::STR))),
   |                                        ^^^^^^^^^^^^^^^^^^
   = note: expected struct `Arc<kclvm_sema::ty::Type>`
              found struct `Rc<kclvm_sema::ty::Type>`
note: expected struct `Arc`, found struct `Rc`
  --> sema/benches/my_benchmark.rs:14:60
   |
14 |                     Rc::new(Type::dict(Rc::new(Type::STR), Rc::new(Type::STR))),
   |                                                            ^^^^^^^^^^^^^^^^^^
   = note: expected struct `Arc<kclvm_sema::ty::Type>`
              found struct `Rc<kclvm_sema::ty::Type>`
note: associated function defined here
  --> /workspaces/kcl/kclvm/sema/src/ty/constructor.rs:59:12
   |
59 |     pub fn dict(key_ty: TypeRef, val_ty: TypeRef) -> Type {
   |            ^^^^

error[E0308]: arguments to this function are incorrect
  --> sema/benches/my_benchmark.rs:16:25
   |
16 |                 Rc::new(Type::dict(Rc::new(Type::ANY), Rc::new(Type::ANY))),
   |                         ^^^^^^^^^^
   |
note: expected struct `Arc`, found struct `Rc`
  --> sema/benches/my_benchmark.rs:16:36
   |
16 |                 Rc::new(Type::dict(Rc::new(Type::ANY), Rc::new(Type::ANY))),
   |                                    ^^^^^^^^^^^^^^^^^^
   = note: expected struct `Arc<kclvm_sema::ty::Type>`
              found struct `Rc<kclvm_sema::ty::Type>`
note: expected struct `Arc`, found struct `Rc`
  --> sema/benches/my_benchmark.rs:16:56
   |
16 |                 Rc::new(Type::dict(Rc::new(Type::ANY), Rc::new(Type::ANY))),
   |                                                        ^^^^^^^^^^^^^^^^^^
   = note: expected struct `Arc<kclvm_sema::ty::Type>`
              found struct `Rc<kclvm_sema::ty::Type>`
note: associated function defined here
  --> /workspaces/kcl/kclvm/sema/src/ty/constructor.rs:59:12
   |
59 |     pub fn dict(key_ty: TypeRef, val_ty: TypeRef) -> Type {
   |            ^^^^
.........
```

### 4. What is your KCL components version? (Required)

latest

--- a/kclvm/sema/benches/my_benchmark.rs
+++ b/kclvm/sema/benches/my_benchmark.rs
@@ -1,19 +1,19 @@
-use std::rc::Rc;
-
 use criterion::{criterion_group, criterion_main, Criterion};
 use kclvm_sema::ty::*;
+
+use std::sync::Arc;
 
 pub fn criterion_benchmark(c: &mut Criterion) {
     c.bench_function("sup", |b| {
         b.iter(|| {
             let types = vec![
-                Rc::new(Type::int_lit(1)),
-                Rc::new(Type::INT),
-                Rc::new(Type::union(&[
-                    Rc::new(Type::STR),
-                    Rc::new(Type::dict(Rc::new(Type::STR), Rc::new(Type::STR))),
+                Arc::new(Type::int_lit(1)),
+                Arc::new(Type::INT),
+                Arc::new(Type::union(&[
+                    Arc::new(Type::STR),
+                    Arc::new(Type::dict(Arc::new(Type::STR), Arc::new(Type::STR))),
                 ])),
-                Rc::new(Type::dict(Rc::new(Type::ANY), Rc::new(Type::ANY))),
+                Arc::new(Type::dict(Arc::new(Type::ANY), Arc::new(Type::ANY))),
             ];
             sup(&types);
         })

